cost/aws/old_snapshots/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.6
```

Comment:
This version bump was not actually included in the actual Policy Template file, so it probably wasn't published.